src/lib/Constants.sol

```diff
@@ -93,4 +95,14 @@ library Constants {
 
     // Morpho
     address public constant MORPHO = 0x33333aea097c193e66081E930c33020272b33333;
+
+    /*//////////////////////////////////////////////////////////////
+                          TESTING ADDRESSES
+    //////////////////////////////////////////////////////////////*/
+
+    // address used for forked node testing
+    address public constant ALICE = 0xe43F4840Dad185BEEF6daA8E7328b521e6a1a2A0;
+
+    // address used for forked node testing
+    address public constant BOB = 0xFCC7e03C00FC57d964AEb22029Df7AD6eA795d49;
```

Comment:
does this need to be here? almost all contracts inherit this meaning it will be included in all the contracts deployed on main